src/Command/CsvImportCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Product;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputArgument;
+use League\Csv\Reader;
+
+class CsvImportCommand extends Command
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $em;
+
+    /**
+     * CsvImportCommand constructor.
+     *
+     * @param EntityManagerInterface $em
+     *
+     * @throws \Symfony\Component\Console\Exception\LogicException
```

Comment:
do not write the full path. 
use Symfony\Component\Console\Exception
@throws LogicException